openlibrary/macros/OLID.html

```diff
@@ -0,0 +1,15 @@
+$def with (q="")
+
+$ edition = query_param('edition', q)
```

Comment:
The call to the macro should be providing an `ocaid` so there will not be a query_param here. A query param is a thing passed in to the url, e.g. `openlibrary.org?param=123`